Yank/utils.py

```diff
@@ -845,6 +855,170 @@ def camelcase_to_underscore(camelcase_str):
     underscore_str = re.sub(r'([A-Z])', '_\g<1>', camelcase_str)
     return underscore_str.lower()
 
+def quantity_from_string(quantity_str):
```

Comment:
Yeah, it is probably worth to write a simple test in which you throw few expressions at it in case anybody will end up using its advanced functionalities.